torch/csrc/jit/python/script_init.cpp

```diff
@@ -788,6 +798,11 @@ void initJitScriptBindings(PyObject* module) {
                   return method.name();
                 });
               })
+          .def(
+              "_properties", [](Object& self) { return self.get_properties(); })
+          .def(
+              "equals",
```

Comment:
should we just consider overriding `__eq__` so that comparison in python is more natural?